app/views/selectFile.html

```diff
@@ -6,7 +6,7 @@
     <div class="form-group-horizontal">
         <p class="note-required">* indicates required field</p>
 
-        <p ng-if="errors.global" class="page-errors">{{errors.global}}</p>
+        <p ng-if="errors.global" class="page-errors" role="alert">{{errors.global}}</p>
```

Comment:
If this needs `role="alert"`, the login partial probably needs modified to include the role for the login error as well, yes?